master/master.go

```diff
@@ -33,6 +37,13 @@ func TerminateSlaves(svc *ec2.EC2, instances []*ec2.Instance) (*ec2.TerminateIns
 	return svc.TerminateInstances(params)
 }
 
+func SendTask(t *task.Task, i *ec2.Instance) (resp *http.Response, err error) {
+	url := "http://" + *i.PublicIpAddress + ":8080/tasks/create"
```

Comment:
Can we perhaps create a folder to put all the common/shared types and strings (e.g. the resource names `/tasks/create`, the types for {de,}serialising JSON)? For example lib/common.go. Most Go projects do this because Go doesn't do circular imports so if we have types in two packages and they each import the other then we'll have problems.